docs_src/roadmap.md

```diff
@@ -5,4 +5,89 @@
+          The time taken to operationalize new AI-based software in a new environment is often long. Working with our partners, we would focus our efforts in reducing the time to production.</li>
+        </ul>
+      </p>
+     <p><strong>Out-of-Scope</strong></p>
+      There are several items that will not be considered as part of this reference implementation. This is not an exhaustive list, but these are core exclusions as they are not our differentiators:
+      <ul>
+         <li>We will not build or recommend AI models</li>
+         <li>We will not build end-to-end solutions</li>
+         <li>We will not advocate for a specific software deployment architecture</li>
+      </ul>   
+    </td>
+  </tr>
+  <tr>
+    <td><strong>Next</strong>
+      <ul>
+        <li><strong>Distributed Architecture</strong><br>
+        Separating out media pre-processing, AI inferencing and post-processing as completely independent services. Providing a mechanism to deploy and run the individual services across distributed heterogeneous compute. Publish all events to an Enterprise Service Bus (ESB).</li>
+        <li><strong>Update Benchmark script</strong><br>
+        Update the script to reflect the new distributed architecture.</li>
+        <li><strong>Benchmark Results</strong><br>
+        Benchmark results across CPU and GPU and GPU only hardware SKUs (previous results updated and new SKUs added)</li>
+      </ul>
+    </td>
+  </tr>
+  <tr>
+    <td><strong>Later</strong>
+      <ul>
+        <li><strong>Model Drift and updates</strong><br>
+        Monitor model accuracy and rectify model drift. Add new models in live systems.</li>
+        <li><strong>Edge Training</strong>
+        Provide a mechanism to do localized instore training to ensure new products are identified without delay.</li>
+        <li><strong>Hierarchical model support</strong><br>
+        Provide a mechanism to have a hierarchy of models that gets chosen and executed based on initial segmentation.</li>
+        <li><strong>Dev Cloud Support</strong><br>
+        Provide a cloud environment for partners to access the hardware and run benchmarks.</li>
+        <li><strong>Camera Management</strong><br>
+        Provide a mechanism to onboard and drive the camera lifecycle across the store.</li>
+      </ul>
+    </td>
+  </tr>
+</tbody>
+</table>
+
+**Disclaimer**: The roadmap is for informational purposes only and is subject to change.
+  
+**Help Drive Our Roadmap Priorities**
+We want to drive our roadmap by building the most valuable assets/ingredients to our partners. Our roadmap will be public and will provide complete transparency to ensure we receive continuous feedback from our partners. Raise any issues and requirements to help us guide our roadmap priorities, and in doing so we can drive the retail vertical forward. 
+
+Open an [issue on GitHub](https://github.com/intel-retail/automated-self-checkout/issues) to report a problem or to provide feedback.
+
```

Comment:
Farhaan wanted the rows to be merged on the second column. It wasn't possible in Markdown. I have added a raw HTML table, which is being rendered in .md file. Please delete it if it doesn't make sense.